org.scala-ide.sdt.aspects/plugin.xml

```diff
@@ -8,4 +8,5 @@
    <extension-point id="cuprovider" name="Compilation Unit Provider" schema="cuprovider.exsd"/>
    <extension-point id="imagedescriptorselector" name="Image Descriptor Selector" schema="imagedescriptorselector.exsd"/>
    <extension-point id="openactionprovider" name="Open Action Provider" schema="openactionprovider.exsd"/>
+   <extension-point id="lncheck" name="Line Number Check" schema="lncheck.exsd"/>
```

Comment:
I wanted to squash it but every time I didn't see this commit in 'rebase' editor.